_pages/documentation/developer/server-permissions.html.twig

```diff
@@ -0,0 +1,91 @@
+---
+section: documentation
+sectiontitle: Developer Documentation
+permalink: /documentation/developer/server-permissions/
+title: Server Permissions
+parent:
+---
+
+{% extends "_layouts/page.html.twig" %}
+
+{% from _self import slash_command_link %}
+
+{% macro slash_command_link(permission) %}
+    {% if permission['no-link'] %}
+        /{{ permission['slash-command'] }}
+    {% else %}
+        {% set fragment = permission['html-id'] ? '#' ~ permission['html-id'] : '' %}
+
+        <a href="{{ url(collections.slash_commands[permission['slash-command']]) }}{{ fragment }}">
+            /{{ permission['extended-command'] ?? permission['slash-command'] }}
+        </a>
+    {% endif %}
+{% endmacro %}
+
+{% block content %}
+    <h1>Server Permissions</h1>
+    <p class="mb-3">
+        Permissions can be assigned to groups and individual players on a server. These permissions control what a player may or may not do on the server. Many of the permissions control which <a href="/documentation/slash-commands/">commands</a> can be used by players.
+    </p>
+    <table>
+        <thead>
+            <tr>
+                <th>Permission</th>
+                <th>Description</th>
+            </tr>
+        </thead>
+        <tbody>
+        {% for permission in data.server_permissions if permission.description %}
+            <tr>
+                <td>{{ permission.permission }}</td>
+                <td>{{ permission.description | markdown}}</td>
+            </tr>
+        {% endfor %}
+        </tbody>
+    </table>
+
+    <h2 id="commands">Commands</h2>
```

Comment:
Assuming you agree with my idea of linking permissions on the slash commands page, then this should move there?

Reply:
I think keeping a "Commands" section here is ok because if someone is reading this documentation when building their groups file, they'd want to be able to see what slash commands are being given according to the permissions.